app/src/main/java/com/kamron/pogoiv/ScreenGrabber.java

```diff
@@ -58,6 +62,35 @@ public static ScreenGrabber getInstance() {
         return instance;
     }
 
+    @NonNull private static Rect getBitmapBounds(Bitmap bmp) {
+        return new Rect(0, 0, bmp.getWidth(), bmp.getHeight());
+    }
+
+    static @Nullable @ColorInt int[] getPixelsSafe(Bitmap bmp, Point[] points) {
```

Comment:
Those crashes were fixed by checking if the image was portrait/landscape, but we still got further crashes (though, it turns out, not here): http://crashes.to/s/c9a025fd5e9. Opened #420 to track that.